lib/scanner.php

```diff
@@ -24,7 +24,6 @@
 namespace OCA\Files_Antivirus;
 
 use OCP\IL10N;
-use OCA\Files_Antivirus\Item;
 
 class Scanner {
```

Comment:
If I understand correctly this class functions both as factory/wrapper and base class.

I would make this class either and interface or an abstract class and put the factory logic in a new class